inside-filters/adblock_wewnetrzne.txt

```diff
@@ -1,20 +1,24 @@
-[Adblock Plus 2.0]
 ! Title: Polskie filtry wewnętrzne
-! Collaborators: mat41997, MajkiIT, hawkeye116477
+! Collaborators: KonoromiHimaries, MajkiIT, hawkeye116477
 ! Homepage: https://certyficate.it
-! EULA: Instalacja|Użytkowanie listy "Polskie filtry wewnętrzne" jest równoznaczne z akceptacją co do bycia ich komitentem, a twórcy nie ponoszą odpowiedzialności prawnej za nadużycia użytkowników.
-! Last modified: 31 December 2017, 16:43
-! Version: 116
+! Statement: https://github.com/MajkiIT/polish-ads-filter/wiki/Og%C3%B3lna-Koncepcja
+! EULA: Subskrybcja|Użytkowanie listy "Polskie filtry wewnętrzne" jest równoznaczne z akceptacją co do bycia ich komitentem, a twórcy nie ponoszą odpowiedzialności prawnej za nadużycia użytkowników.
+! Last modified: 5 January 2018, 19:57
+! Version: 125
 ! Expires: 3 days
-! Support: 
+! Wymagane:
+!   Rozszerzenia >> uBlock Origin / Adguard lub ich aktualne forki
```

Comment:
Powinno być raczej AdGuard :smile:.

Reply:
![2018-01-05 20_55_56-bloker reklam adguard - chrome web store](https://user-images.githubusercontent.com/5884000/34625953-08ce8008-f25b-11e7-8a13-96266128d352.png)

Reply:
A tutaj jest znowu inaczej -> https://itunes.apple.com/us/app/adguard-adblock-privacy/id1047223162?mt=8. Na stronie głównej AG tak samo -> https://adguard.com/pl/welcome.html :smile:.

Reply:
Z tego co wiem, to powinno być `AdGuard`